src/components/TabItem/TabItem.jsx

```diff
@@ -2,11 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './TabItem.styl';
 
-const TabItem = ({ title, isActive = false }) => <div className={`${styles.TabItem} ${isActive ? styles.TabItemActive : ''}`}>{title}</div>;
+const TabItem = ({ id, title, isActive = false, onClick = null }) => {
+  return (
+    <div
+      className={`${styles.TabItem} ${isActive ? styles.TabItemActive : ''}`}
+      onClick={() => { typeof onClick === 'function' ? onClick(id, title) : void 0} }
```

Comment:
you can pass `noop` as default value to `onClick` -> `onClick={() => { onClick(id, title) }`

or 

this one should work fine as well I think `onClick={() => { onClick && onClick(id, title) }`